docs/source/reference/respawn/damagehistory.rst

```diff
@@ -0,0 +1,114 @@
+Damage History 
+==============
+
+After a player receives damage it is stored by the game in an array of the following struct. The information in that struct is final and changing it does not affect the game.
+
+DamageHistoryStruct
+-------------------
+
+.. cpp:struct:: DamageHistoryStruct
+    
+    .. cpp:var:: string attackerName
+	
+        name of the attacker 
```

Comment:
What's up with the inconsistent casing btw? This starts lowercase but later it starts uppercase?

Reply:
Well.... its called "not caring" will change later

Reply:
> will change later

Also known as the greatest lie in software development :P